.github/workflows/main.yml

```diff
@@ -70,6 +70,9 @@ jobs:
       - name: Build assets
         run: just assets-build
 
+      - name: Collect assets
+        run: just collectstatic
```

Comment:
We can flatten these two into the `just assets-install` step, and use `just assets` there, it does install, build, and collectstatic in one place then?